src/wallet/rpcwallet.cpp

```diff
@@ -3709,52 +3709,57 @@ UniValue getaddressinfo(const JSONRPCRequest& request)
-            "  \"labels\"                      (object) Array of labels associated with the address.\n"
+            "  \"sigsrequired\" : xxxxx              (numeric, optional) The number of signatures required to spend multisig output (only if script is multisig).\n"
+            "  \"pubkey\" : \"publickeyhex\",          (string, optional) The hex value of the raw public key for single-key addresses (possibly embedded in P2SH or P2WSH).\n"
+            "  \"embedded\" : {...},                 (object, optional) Information about the address embedded in P2SH or P2WSH, if relevant and known. Includes all\n"
+            "                                                         getaddressinfo output fields for the embedded address, excluding metadata (timestamp, hdkeypath,\n"
+            "                                                         hdseedid) and relation to the wallet (ismine, iswatchonly).\n"
+            "  \"iscompressed\" : true|false,        (boolean, optional) If the pubkey is compressed.\n"
+            "  \"label\" :  \"label\"                  (string) The label associated with the address. Defaults to \"\". Equivalent to the name field in the labels array.\n"
+            "  \"timestamp\" : timestamp,            (number, optional) The creation time of the key if available, expressed in seconds since Epoch Time (Jan 1 1970 GMT).\n"
+            "  \"hdkeypath\" : \"keypath\"             (string, optional) The HD keypath, if the key is HD and available.\n"
+            "  \"hdseedid\" : \"<hash160>\"            (string, optional) The Hash160 of the HD seed.\n"
+            "  \"hdmasterfingerprint\" : \"<hash160>\" (string, optional) The fingerprint of the master key.\n"
+            "  \"labels\"                            (object) An array of labels associated with the address. Currently limited to one label but returned\n"
+            "                                               as an array to keep the API stable if multiple labels are enabled in the future.\n"
             "    [\n"
             "      { (json object of label data)\n"
-            "        \"name\": \"labelname\" (string) The label\n"
-            "        \"purpose\": \"string\" (string) Purpose of address (\"send\" for sending address, \"receive\" for receiving address)\n"
+            "        \"name\": \"label name\" (string) The label name. Defaults to \"\". Equivalent to the label field above.\n"
+            "        \"purpose\": \"purpose\" (string) The purpose of the associated address (send or receive).\n"
             "      },...\n"
             "    ]\n"
             "}\n"
                 },
                 RPCExamples{
-                    HelpExampleCli("getaddressinfo", "\"1PSSGeFHDnKNxiEyFrD1wcEaHr9hrQDDWc\"")
-            + HelpExampleRpc("getaddressinfo", "\"1PSSGeFHDnKNxiEyFrD1wcEaHr9hrQDDWc\"")
+                    HelpExampleCli("getaddressinfo", "\"bc1q09vm5lfy0j5reeulh4x5752q25uqqvz34hufdl\"") +
```

Comment:
Is this intentionally an invalid address? The previous example was a valid address, which I think is useful since users can test the RPC and see a valid return object by using the example

Reply:
Yes, I noticed addresses with a missing character in other RPC examples and thought it might be the norm. Will update #17578.

Reply:
Added a valid address in #17578.